x-pack/legacy/plugins/siem/public/components/embeddables/map_config.ts

```diff
@@ -7,6 +7,32 @@
 import uuid from 'uuid';
 import { IndexPatternMapping } from './types';
 
+// Update source/destination field mappings to modify what fields will be returned to map tooltip
+const sourceFieldMappings: Record<string, string> = {
+  'host.name': 'Host',
```

Comment:
is i18n required here (for the values `Host`, `Source IP`, etc, and in the `destinationFieldMappings` const below)?

Reply:
Nice catch! And that's why you should _always_ write your i18n the same time you write any user facing string, ha! 🙂